examples/stable_diffusion_v2/conditions/openpose/hand.py

```diff
@@ -0,0 +1,83 @@
+import cv2
+import numpy as np
+from scipy.ndimage import gaussian_filter
+from skimage.measure import label
+
+import mindspore as ms
+
+from . import util
+from .model import handpose_model
+
+
+class Hand(object):
+    def __init__(self, model_path):
+        self.model = handpose_model()
+        model_dict = ms.load_checkpoint(model_path)
+        param_not_load, ckpt_not_load = ms.load_param_into_net(self.model, model_dict)
+        self.model.set_train(False)
+        self.model.set_train(False)
+
+    def __call__(self, oriImg):
+        scale_search = [0.5, 1.0, 1.5, 2.0]
```

Comment:
Why is the` scale_search` different from the `scale_search` in bodypose model?

Reply:
Maybe this setting is best performance. Just migrate it from the official repo.